cmdpolling/continuousreport.go

```diff
@@ -0,0 +1,96 @@
+package cmdpolling
+
+import (
+	"errors"
+	"fmt"
+	"time"
+	"os"
+	"encoding/json"
+
+	log "github.com/Sirupsen/logrus"
+	"github.com/codegangsta/cli"
+	"github.com/ingrammicro/concerto/api/types"
+	"github.com/ingrammicro/concerto/cmd"
+	"github.com/ingrammicro/concerto/utils"
+	"github.com/ingrammicro/concerto/utils/format"
+)
+
+const (
+	RetriesNumber = 5
+	RetriesFactor = 3
+)
+
+func cmdContinuousReportRun(c *cli.Context) error {
+	log.Debug("cmdContinuousReportRun")
+
+	formatter := format.GetFormatter()
+	pollingSvc := cmd.WireUpPolling(c)
+
+	// cli command argument
+	var cmdArg string
+	if c.Args().Present() {
+		cmdArg = c.Args().First()
+	} else {
+		formatter.PrintFatal("argument missing", errors.New("a script or command is required"))
+	}
+
+	// cli command thresholds flags
+	thresholds := utils.Thresholds{Lines: c.Int("lines"), Time: c.Int("time"), Bytes: c.Int("bytes")}
+
+	// Custom method for chunks processing
+	fn := func(chunk string) error {
```

Comment:
It may make sense to convert this closure into a (named) function (that takes the pollingSvc as parameter), because its logic is not trivial

Reply:
I thought about that but due to some integration issues so i declined that way in previous days. But i would like to implement it